x-pack/plugins/security_solution/common/api/detection_engine/rule_management/index.ts

```diff
@@ -30,5 +30,6 @@ export * from './import_rules/rule_to_import_validation';
 export * from './import_rules/rule_to_import';
 export * from './model/query_rule_by_ids_validation';
 export * from './model/query_rule_by_ids';
+export * from './preview_rules/preview_rules_route';
```

Comment:
rule preview route is a separate sub-domain in detection_engine folder structure, why not preserve this for schemas?
preview also historically belongs to other team code area ownership, then rule management APIs

Reply:
good point, split into separate `rule_preview` domain :+1: